lib/spec/runners/epoch_processing.ex

```diff
@@ -66,4 +66,19 @@ defmodule EpochProcessingTestRunner do
         assert Diff.diff(state, post) == :unchanged
     end
   end
+
+  defp handle_case("participation_flag_updates", pre, post) do
```

Comment:
Please remove all `handle_case` except the first one (i.e. `defp handle_case(name, pre, post) do ...`).